controllers/admin/AdminStoresController.php

```diff
@@ -283,13 +283,13 @@ public function renderForm()
         }
 
         $days = array();
-        $days[1] = $this->trans('Monday', array(), 'Admin.ShopParemeters.Feature');
-        $days[2] = $this->trans('Tuesday', array(), 'Admin.ShopParemeters.Feature');
-        $days[3] = $this->trans('Wednesday', array(), 'Admin.ShopParemeters.Feature');
-        $days[4] = $this->trans('Thursday', array(), 'Admin.ShopParemeters.Feature');
-        $days[5] = $this->trans('Friday', array(), 'Admin.ShopParemeters.Feature');
-        $days[6] = $this->trans('Saturday', array(), 'Admin.ShopParemeters.Feature');
-        $days[7] = $this->trans('Sunday', array(), 'Admin.ShopParemeters.Feature');
+        $days[1] = $this->trans('Monday', array(), 'Admin.ShopParameters.Feature');
```

Comment:
I'm pretty sure days of week are already translated  in ``Admin.Global`` domain..